cli/functionary/client.py

```diff
@@ -94,6 +87,10 @@ def _send_request(endpoint, request_type, post_data=None, post_files=None):
 
     if response.ok:
         return response
+    elif response.status_code == 400:
+        raise click.ClickException(
+            "Please set an active environment id using 'environment set'"
```

Comment:
For right now I think we can just go with this and assume that a 400 is because the header is missing.  In reality, a 400 could represent a whole host of things.  Since the errors returned don't currently include a good, parsable error code (you get the HTTP status code of 400, and then some human friendly message), there's not a good way to discern why the 400 was returned here.  So keep this as is for now and at some point when the API returns errors codes we can come back and update this handling.

Maybe put a comment here explaining that:

```python
# TODO: Add checking of error codes in addition to status once they are added to the API
```